config/prod/ecmonsen-emorypipeline-infra.yaml

```diff
@@ -0,0 +1,22 @@
+template_path: "remote/ecmonsen-emorypipeline-infra.yaml"
+stack_name: "ecmonsen-emorypipeline-infra"
+dependencies:
+  - prod/ecmonsen-emorypipeline-ci.yaml
+parameters:
```

Comment:
seems to be missing SynapseConfig and SourceAccountId?

Reply:
SourceAccountId is in there.

I removed SynapseConfig from the CFN template because it always has a default value. A human will have to update the secret synapse config manually after this is deployed to AWS.

Reply:
(The CFN template where I removed SynapseConfig is in [ecmonsen-emorypipeline-lambda-code](https://github.com/Sage-Bionetworks/ecmonsen-emorypipeline-lambda-code)